pango/Gir.toml

```diff
@@ -206,10 +203,26 @@ boxed_inline = true
 name = "Pango.Context"
 status = "generate"
     [[object.function]]
+    name = "new"
+        # Can be removed once https://gitlab.gnome.org/GNOME/pango/-/merge_requests/651
+        # has been merged.
+        [object.function.return]
+        nullable = false
+    [[object.function]]
+    name = "get_language"
+        # Can be removed once https://gitlab.gnome.org/GNOME/pango/-/merge_requests/651
```

Comment:
You don't change that one there. I think that needs a separate issue to first of all figure out how this function is supposed to work.

Reply:
Done.